Pocket/Pocket/PocketRequestManager.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PocketRequestManager.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 6/7/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public class PocketRequestManager: NSObject, PocketRequestManagerProtocol {
```

Comment:
Why does this need to extend NSObject? Make this a struct that conform to your protocol.